WordPress/Classes/ViewRelated/Stats/Helpers/StatSection.swift

```diff
@@ -35,7 +35,7 @@
     case postStatsRecentWeeks
 
     static var allInsights: [StatSection] {
-        if FeatureFlag.statsNewInsights.enabled {
+        if AppConfiguration.isJetpack {
```

Comment:
I'm not sure AppConfiguration.isJetpack since WordPress simply won't have access to the stats screen. This comment applies to the following commit: https://github.com/wordpress-mobile/WordPress-iOS/pull/21499/commits/a80901ed0e006cc3cbac3bf6d7707eec09f0c854.

Reply:
Fixed! https://github.com/wordpress-mobile/WordPress-iOS/pull/21499/commits/71bdd988bbf4dd34e7b20e22f2e3bc92ff3e218d